src/styles/components/main.scss

```
@@ -0,0 +1,25 @@
+.main {
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  gap: 46px 48px;
+
+  align-items: center;
+
+  align-content: center;
+  justify-content: center;
+  padding: 50px 40px;
+}
```

It's better to struct at first general styles and then write media, it will be more readable. And you don't need here align-items: center;

```suggestion
.main {
  display: grid;
  grid-template-columns: repeat(1, 200px);
  gap: 46px 48px;
  align-content: center;
  justify-content: center;
  padding: 50px 40px;

  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
}
```